Description: This patch fixes the destination directory of nodejs files
Author: Sebastien Badia <seb@sebian.fr>
Forwarded: not-needed
Last-Update: 2014-05-17

Index: nodejs-pkg/tools/install.py
===================================================================
--- nodejs-pkg.orig/tools/install.py	2014-05-17 02:06:32.000000000 +0000
+++ nodejs-pkg/tools/install.py	2014-05-17 02:12:57.904060574 +0000
@@ -213,8 +213,8 @@
   target_defaults = conf['target_defaults']
 
   # argv[2] is a custom install prefix for packagers (think DESTDIR)
-  dst_dir = node_prefix = variables.get('node_prefix') or '/usr/local'
-  if len(args) > 2: dst_dir = abspath(args[2] + '/' + dst_dir)
+  dst_dir = node_prefix = variables.get('node_prefix') or '/usr'
+  if len(args) > 2: dst_dir = abspath(args[2] + '/usr')
 
   cmd = args[1] if len(args) > 1 else 'install'
   if cmd == 'install': return files(install)
